client_admin/pulp/client/admin/auth.py

```diff
@@ -82,9 +82,10 @@ def __init__(self, context):
         # List Command
         list_command = PulpCliCommand('list', 'lists summary of users registered to the Pulp server', self.list)
         list_command.add_option(PulpCliFlag('--details', 'if specified, all the user information is displayed'))
-        list_command.add_option(PulpCliOption('--fields', 'comma-separated list of user fields; if specified, '
-                                                          'only the given fields will be displayed. '
-                                                          'Example: "login,name."', required=False))
+        list_command.add_option(PulpCliOption('--fields', 'comma-separated list of user fields; '
+                                                          'Example: `login,name`. If specified, '
```

Comment:
Only use the backtick escaping if it's in a .rst file. Since this will be shown on the commandline backticks mean something else. Use " here. Sorry for the confusion.